setup/module/api/policies/elb_group_policy.json

```diff
@@ -4,7 +4,30 @@
 	{
             "Effect": "Allow",
             "Action": [
-                "elasticloadbalancing:Describe*"
+                "elasticloadbalancing:ModifyListener",
+                "elasticloadbalancing:CreateLoadBalancer",
+                "elasticloadbalancing:RegisterTargets",
+                "elasticloadbalancing:CreateTargetGroup",
+                "elasticloadbalancing:*",
```

Comment:
Do we need `*` when we already reference all these other actions?

Reply:
✅